oak-lucene/src/test/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneFacetCommonTest.java

```diff
@@ -46,13 +44,6 @@ protected Repository createJcrRepository() {
         return jcr.createRepository();
     }
 
-    @Override
-    @Test
-    @Ignore("failing in lucene only, needs more investigation")
```

Comment:
I think best practise is to use the oak issue in the Ignore annotation ("OAK-9912"?)

Reply:
🤦‍♂️  (facepalm)